libs/sdk-common/Cargo.toml

```diff
@@ -29,6 +29,8 @@ tonic = { workspace = true, features = [
     "tls-webpki-roots",
 ] }
 url = "2.5.0"
+elements = "0.24.1"
```

Comment:
Better to make the elements dependency as optional under feature as well and list that feature as part of  "liquid" features list.

Reply:
Done.

Build elements crate only as part of liquid feature 7debe215449fbca9eaa294cf3f969391c5c9897c

Reply:
I'm going through the PR and I've also added `urlencoding` to the optional dependency list.

- Build urlencoding crate only as part of liquid feature 72ca6efb1650fa408dbb9b5f6db9b16664943047